gemfiles/rails_3_2.gemfile.lock

```diff
@@ -0,0 +1,60 @@
+PATH
```

Comment:
Do we need to have these Gemfile.lock files under source control? Perhaps Bundler could generate them on test run.